src/main/java/com/hubspot/jtopia/TermsExtractor.java

```diff
@@ -0,0 +1,131 @@
+package com.hubspot.jtopia;
```

Comment:
I don't think you need to repackage this - this class could be left in the old package. This has the advantage that it's much clearer to see which changes you made yourself - right now this just shows up as a whole file create and delete. Same for the other repackaged classes.